.gitignore

```diff
@@ -1,4 +1,5 @@
 create_database.py
```

Comment:
please fix problems, then do some actions like registering new user, viewing all cards etc
and attach logs for each action please

Reply:
You need to show me how should I attach logs properly